### PR TITLE
Update user guide and index.md

### PR DESCRIPTION
**Pull request by ys112**
UI screenshot has been updated.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -75,18 +75,18 @@ Shows a message explaining how to access the help page.
 
 Format: `help`
 
-### Add an applicant: `add_applicant`
+### Add an applicant: `add`
 
 Adds an applicant to HRConnect.
 
-Format: `add_applicant /name Name /phone Phone_Number /email Email /address Address /role Role [/tag Tag]…​`
+Format: `add /name Name /phone Phone_Number /email Email /address Address /role Role /note Note [/tag Tag]…​`
```
Comment (xinen26) [inline docs/UserGuide.md:82]: I think /note Note has to be in brackets because iirc its optional

File: docs/UserGuide.md [hunk 2]
```diff
@@ -218,7 +217,7 @@ Furthermore, certain edits can cause the HRConnect to behave in unexpected ways
 
 Action | Format, Examples
 --------|------------------
-**Add_applicant** | `add_applicant /name Name /phone Phone_Number /email Email /address Address /role Role [/tag Tag]…​` <br> e.g., `add_applicant /name James Chow /phone 96622612 /email james@example.com /address 321, Clementi Ave 2, #02-25 /role Junior Engineer`
+**Add** | `add /name Name /phone Phone_Number /email Email /address Address /role Role /note Likes to swim [/tag Tag]…​` <br> e.g., `add /name James Chow /phone 96622612 /email james@example.com /address 321, Clementi Ave 2, #02-25 /role Junior Engineer /note Likes to swim`
```
Comment (xinen26) [inline docs/UserGuide.md:220]: i think maybe should put /note Note instead

**Review by xinen26 — COMMENTED** [review]
Looks good to me! Just a few nits mentioned below

File: docs/UserGuide.md (hunk 1 above)
Comment (ys112) [reply docs/UserGuide.md:82]: Good catch thanks!

**Review by xinen26 — APPROVED** [review]

**Pull request merged**